settings.gradle.kts

```diff
@@ -16,3 +18,7 @@ pluginManagement {
                     .split("+")[0] // Ensures sentences like `+build.1` are ignored
     }
 }
+
+rootDir.listFiles { f: File -> f.isDirectory && f.name.startsWith("kilt-") }?.forEach { dir ->
+    include(dir.name)
+}
```

Comment:
I would prefer if this was a manual detection rather than an automatic detection.
Or, otherwise, place them into a "modules" folder for better organization.

Reply:
Which would you prefer?

Reply:
The "modules" folder would definitely be better, since that handles automatically adding them without needing to filter by a specific prefix.

In which case, I think `kilt-modlauncher` would work better as `modules/modlauncher`.